src/js/components/Calendar/doc.js

```diff
@@ -43,7 +43,16 @@ export const doc = Calendar => {
       dates, and \`isSelected\` is a boolean indicating whether this date
       is selected.`,
     ),
-    date: PropTypes.string.description('The selected date in ISO8601 format'),
+    date: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.arrayOf(
+        PropTypes.oneOfType([
+          PropTypes.string,
+          PropTypes.arrayOf(PropTypes.string),
+        ]),
+      ),
+    ]).description(`The selected date in ISO8601 format. You can also pass
```

Comment:
Suggesting we simplify to: The selected date or array of dates in ISO8601 format.